core/blockchain.go

```diff
@@ -775,31 +775,37 @@ func (self *BlockChain) reorg(oldBlock, newBlock *types.Block) error {
 }
 
 func (self *BlockChain) update() {
-	events := self.eventMux.Subscribe(queueEvent{})
+	// Subscribe to blockchain insertion event sets
+	eventCh := self.eventMux.Subscribe(queueEvent{}).Chan()
```

Comment:
I see no need for a "later refactor". The bug is on develop anyway, fixing it on develop won't make anyone any happier. Fix up PR :)